mainWindow.py

```diff
@@ -25,59 +25,62 @@ def __init__(self, parent=None):
 
         self.sourceModel = QStandardItemModel(self.sourceListView)
         self.sourceListView.setModel(self.sourceModel)
-
+        
         self.t1Series = []
 
+        #Add options to dataTypeComboBox
+        self.dataTypeComboBox.addItems(["Dixon Format","Wash U v1","Wash U v2"])
```

Comment:
I would add these strings from Designer. Right-click the combo box in Qt Designer and select Edit Items. Then enter the items

Reply:
Thanks for the tip!